docs/src/components/ReducedMotionWarning/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './styles.module.css';
+
+import { ThemeClassNames, useColorMode } from '@docusaurus/theme-common';
+import Danger from '/static/img/danger.svg';
+import DangerDark from '/static/img/danger-dark.svg';
+
+export default function ReducedMotionWarning() {
+  const { colorMode } = useColorMode();
+  return (
+    <div className={styles.badge}>
+      <div className={styles.dangerMark}>
+        {colorMode === 'light' ? <Danger /> : <DangerDark />}
+      </div>
+      Look like reduced motion is turned on in your system preferences, some of
+      the animations may be skipped.
```

Comment:
```suggestion
      It looks like reduced motion is turned on in your system preferences. Some of
      the animations may be skipped. [Learn more](https://docs.swmansion.com/react-native-reanimated/docs/guides/accessibility#reduced-motion-in-animations).
```

Reply:
There should be no "the" before "reduced motion", should it?

Reply:
I'm not a native speaker but Grammarly says it's fine without the "the"